serving-sdk/pom.xml

```diff
@@ -33,6 +33,12 @@
   </properties>
 
   <dependencies>
+    <!-- https://mvnrepository.com/artifact/com.google.code.gson/gson -->
```

Comment:
Please bump the version number on line 7

Reply:
Done